addons/thanos/0.3.x/thanos-1.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: thanos
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: thanos
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.3.14-1"
+    appversion.kubeaddons.mesosphere.io/thanos: "0.3.14"
+    endpoint.kubeaddons.mesosphere.io/thanos: "/ops/portal/thanos"
+    docs.kubeaddons.mesosphere.io/thanos: "https://thanos.io/getting-started.md/"
+    values.chart.helm.kubeaddons.mesosphere.io/thanos: "https://raw.githubusercontent.com/banzaicloud/banzai-charts/4a2e5274e8149c5c7d02583a6f2f109663c1b94e/thanos/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: true
```

Comment:
```suggestion
  cloudProvider:
    - name: aws
      enabled: true
    - name: azure
      enabled: true
    - name: gcp
      enabled: true
    - name: docker
      enabled: false
    - name: none
      enabled: true
```

Reply:
maybe we should disable it for all others except for `aws` as long as we do not provide permission details for `azure`, `gcp` and `none`

Reply:
Thanos has plenty of backends [documented](https://github.com/thanos-io/thanos/blob/master/docs/storage.md#configuration), so let me disagree here that we need to disable all other cloud providers ;)
I'll paste the link with the docs into [our usage example doc](https://github.com/mesosphere/sre-konvoy-deployers/pull/22/commits/6a969bc002d973846b16964bcec17dca247c6cab)

Reply:
but would you please add then `gcp` 😬